magma_access_gateway_installer/__init__.py

```diff
@@ -25,42 +26,44 @@
 
 
 def main():
-    args = cli_arguments_parser(sys.argv[1:])
-    validate_args(args)
-
     network_interfaces = netifaces.interfaces()
     network_interfaces.remove("lo")
 
+    args = cli_arguments_parser(sys.argv[1:])
+    validate_args(args, network_interfaces)
+
```

Comment:
What's the reason behind moving the arg parsing after removing the `lo` network interface?

Reply:
No specific reason. The only thing that matters is to have `network_interfaces` list available for args validation. 
I just moved the whole block.